server/models/workspaceModel.js

```diff
@@ -60,5 +60,6 @@ const workspaceSchema = mongoose.Schema(
 );
 
 const Workspace = mongoose.model("Workspace", workspaceSchema);
+const Board = mongoose.model("Board", boardSchema);
 
```

Comment:
Workspace should be the default export

Reply:
When we change this, we need to change the imports in the file to `import Workspace from "../models/workspaceModel.js";`